src/makeCalculator.js

```diff
@@ -4,7 +4,43 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add(value) {
+      result += value;
+    },
+
+    subtract(value) {
+      result -= value;
+    },
+
+    multiply(value) {
+      result *= value;
+    },
+
+    divide(value) {
+      if (value !== 0) {
+        result /= value;
+      }
```

Comment:
[CODE KNOWLEDGE] - It's good that you're checking for division by zero, but consider throwing an error or returning a specific value to indicate the error to the user, as silent failure can lead to hard-to-debug issues.